packages/client/src/utils/draftUtils.ts

```diff
@@ -34,7 +34,14 @@ const getInformantEngName = (
       sectionData.firstNamesEng ?? ''
     }`
   }
```

Comment:
How should we handle `lastNameFirst` with middleName? Should it be `last first middle` or `last middle first`

Reply:
@Zangetsu101 it should be last first middle